custom_components/tuya_local/devices/probreeze_pbd23w_dehumidifier.yaml

```diff
@@ -67,45 +59,57 @@ entities:
       - id: 5
         type: boolean
         name: switch
-
   - entity: switch
-    name: LED Display
-    icon: "mdi:led-on"
+    translation_key: sleep
     category: config
     dps:
       - id: 102
         type: boolean
         name: switch
-
-  - entity: switch
-    name: Child Lock
-    icon: "mdi:lock"
+  - entity: lock
+    translation_key: child_lock
     category: config
     dps:
       - id: 7
         type: boolean
-        name: switch
-
+        name: lock
   - entity: binary_sensor
-    name: Defrost Mode
-    class: cold
+    translation_key: defrost
     category: diagnostic
     dps:
       - id: 105
         type: boolean
         name: sensor
-
   - entity: binary_sensor
-    name: Tank Problem
+    translation_key: tank_full
+    dps:
+      - id: 11
+        type: bitfield
+        name: sensor
+        optional: true
+        mapping:
+          - dps_val: 8
+            value: true
+          - value: false
+  - entity: binary_sensor
     class: problem
     category: diagnostic
     dps:
       - id: 11
         type: bitfield
         name: sensor
+        optional: true
         mapping:
+          - dps_val: 0
+            value: false
           - dps_val: 1
             value: false
```

Comment:
1 is not captured anywhere else. It is intentionally excluded from the problem sensor? Or a duplicate of the separate defrost sensor (which is the other "normal operation" flag that dehumidifiers might have in here)

Reply:
I'm not sure what to do here.  `1` seems to be the default state and means "everything is fine".  I haven't seen it be a `0`, but looking at other devices suggests that normally `0` is normal.  So I captured both. 

I suppose `0` should be removed if anything.  

What do you think?